app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,18 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    user_id = params[:user_id]
+    @user = User.find(user_id)
```

Comment:
- [ ] Please set the user to the current session user. Like this: `@user = @current_user`. Or you can create the post as the current user's post like this: `@post = @current_user.posts.new(post_params)`.